packages/components/src/shortened-number/stories/index.stories.jsx

```diff
@@ -1,4 +1,5 @@
 import ShortenedNumber from '..';
+import formttedNumber from './../formattedNumber';
```

Comment:
```suggestion
import formattedNumber from '../formattedNumber';
```

Fixes a typo and an unnecessarily verbose relative directory.